switchboard.py

```diff
@@ -117,7 +117,7 @@ def git_ref(self):
     @property
     def conda_dependencies(self):
         return [
-            "future numpy scipy numba pip nose h5py",
+            "numpy scipy numba pip h5py pytest",
```

Comment:
```suggestion
            "numpy scipy numba pip h5py pytest",
            "-c conda-forge sparse",
```

Reply:
This should fix adding the sparse dependency from conda-forge, note also  that you can run the whole test locally if you want to check it. The instructions are in the README.rst (if they are insufficient or unclear please do let me know and I'll do something about it).